homework2#5.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title></title>
+	<script>
+        
+function addition (a,b){
+    var summary = a+b
```

Comment:
Для лучшей читаемости между оператором и операндом нужен пробел
```suggestion
    var summary = a + b
```